cmake/SickLMS5xxConfig.cmake.in

```diff
@@ -20,10 +20,3 @@ if(NOT TARGET SickLMS5xx::SickLMS5xx)
   include("${CMAKE_CURRENT_LIST_DIR}/SickLMS5xxTargets.cmake")
 endif()
 
-# Have to come last as they modify some variable used above.
-include(CMakeFindDependencyMacro)
-find_dependency(Eigen3 3.3.4 REQUIRED)
-find_dependency(PCL 1.11 REQUIRED COMPONENTS common io)
-# link the deps so that client projects get them also
-target_link_libraries(SickLMS5xx::SickLMS5xx INTERFACE Eigen3::Eigen pcl_common pcl_io)
-target_include_directories(SickLMS5xx::SickLMS5xx INTERFACE $<INSTALL_INTERFACE:include>)
```

Comment:
it make sense to make the PCL links private as you did above, but the generated SickLMS5xxConfig.cmake should still have the `find_dependency()` calls to resolve this stuff at configure time, does including them here lead to any hardcoded paths???

Reply:
i had thought that gets executed when you call find_package(Sick), so there should not be any install paths in there.